data/intro missions.txt

```diff
@@ -451,3 +451,143 @@ conversation "end of intro missions"
 	label next
 	`	There's an awkward pause. You're unsure of how to respond. Then he says, "Thing is, this here planet is one of the most exclusive retirement communities in the galaxy. Plenty of the locals are probably worth more than a billion credits. But they're mostly financial or business types, or old money. I'm rich enough to be here, but I just hope they're willing to socialize with someone who made his fortune flying a ship."`
 	`	He hands you a credit chip worth <payment>, and says, "Best of luck to you, Captain. Don't hesitate to look me up if you're ever in the area."`
+
+
+
+mission "Intro: Follow Up - Hai Before Main Plot"
+	name "Bring James to <planet>"
+	description "Take James to <destination> to meet the Hai."
+	blocked `You consider paying a visit to James to tell him about the Hai, but decide to wait until you have a free seat available on your ship. Just in case.`
+	passengers 1
+	landing
+	source "Hestia"
+	destination "Allhome"
+	to offer
+		or
+			has "Intro [3 Freighter]: done"
+			has "Intro [3 Transport]: done"
+			has "Intro [3 Interceptor]: done"
+		has "First Contact: Hai: offered"
+	to fail
+		has "chosen sides"	
```

Comment:
So you're just kicking James out the airlock the moment you decide to join sides if this mission is active?

Reply:
(he's old anyway)

Reply:
Oh!  I only considered that as a modifier for the start condition.  This can be moved up to a 'not' condition under 'to offer'

Reply:
I think there is some potential to enhance functionality here: 

- Either a timer could be implemented (similar to the university construction chain) where he gets tired and shoves off on his own after a while

- or other missions could occasionally check to see if one of these James-transport jobs is still active and he could interject commentary on whatever the current situation is :D